src/main/java/thrift/logic/commands/BudgetCommand.java

```diff
@@ -35,6 +35,9 @@
 
     public static final String MESSAGE_SUCCESS = "New budget: %1$s";
 
+    public static final String UNDO_SUCCESS = "Budget is reset to: %1$s";
+    public static final String REDO_SUCCESS = "Budget is reset to: %1$s";
```

Comment:
Can specify the month stated in `Budget` object that is undone/redone like:
`"Budget for the month of %s is reset to: %2$s";`

Month can be formatted with `new SimpleDateFormat("MMMM yyyy").format(budget.getBudgetDate());`